src/Razor/Microsoft.AspNetCore.Razor.Tools/src/rzc.csproj

```diff
@@ -3,13 +3,7 @@
   <PropertyGroup>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This assembly contains infrastructure supporting Razor MSBuild integration.</Description>
 
-    <!--
-      This is intentionally targeting netcoreapp3.0. This allows using the Microsoft.NET.Sdk.Razor package without having to solve diamond dependency problems in AspNetCore.
-      We use a runtimeconfig.json.template to allow roll forwards.
-      In addition, the SDK stamps an exact version of the shared fx to use in all runtimeconfig.json that are contained in the SDK.
-      See the discussion here: https://github.com/dotnet/aspnetcore-internal/issues/3201#issuecomment-539631557
-    -->
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
```

Comment:
This is a big shift. @NTaylorMullen and @pranavkm do we really want to move this to work only when targeting 5.0❔ The alternative would be to switch to `netcoreapp3.1`.

Reply:
I already confirmed with @pranavkm

Reply:
Well, @NTaylorMullen said `netcoreapp3.1` in an email earlier today&hellip;

Reply:
Wouldn't net5.0 re-introduce the diamond dependency the comment above introduces?